lib/generators/curly/scaffold/templates/model_presenter.rb.erb

```diff
@@ -0,0 +1,19 @@
+class <%= plural_table_name.capitalize %>::<%= singular_table_name.capitalize %>Presenter < Curly::Presenter
```

Comment:
Since I got rid of the base presenter, I actually did the opposite, and got rid of the module definition.  Assume that is all right as it matches the documentation more closely?